doc/README.md

```diff
@@ -61,57 +46,282 @@ this guide.
     If one of your language-aware fences needs an info string that is not
     already on this list, you may use `text` until the grammar gets added to
     [`remark-preset-lint-node`][].
-
   * Code need not be complete. Treat code blocks as an illustration or aid to
     your point, not as complete running programs. If a complete running program
     is necessary, include it as an asset in `assets/code-examples` and link to
     it.
-* When using underscores, asterisks, and backticks, please use
-  backslash-escaping: `\_`, `\*`, and ``\` ``.
+  * When using underscores, asterisks, and backticks, please use
+    backslash-escaping: `\_`, `\*`, and ``\` ``.
+  * No nesting lists more than 2 levels (due to the markdown renderer).
+  * For unordered lists, use asterisks instead of dashes.
+
+### Document Rules
```

Comment:
```suggestion
### Document rules
```

Reply:
If we were using title case capitalization for headers, then yes. But we don't. Microsoft Style Guide is to use sentence case for headers like this.